core/src/world/place/building/business/inn.rs

```diff
@@ -1,5 +1,5 @@
 use crate::utils::pluralize;
-use crate::world::{Demographics, Place};
+use crate::world::{vocabulary::*, Demographics, Place};
```

Comment:
Wildcard imports are generally frowned on in Rust. The exceptions are the `use super::*` import common to test mods, and the `use xyz::prelude::*` import recommended by a lot of libraries, and even the latter is controversial. The notion is that since the contents of `*` can change over time, it introduces unwarranted fragility/obfuscation to your code.

A more idiomatic structure would be:

```suggestion
use crate::world::{vocabulary, Demographics, Place};
```

with invocations replaced with

```diff
- 0 => any_animal(rng),
+ 0 => vocabulary::any_animal(rng),
```

That's obviously a bit verbose, which maybe circles back to a deeper naming issue. Perhaps renaming the `vocabulary` mod simply to `word` would scan better and be more succinct? This is ultimately going to be invoked from dozens of other mods, so it's worth streamlining the ergonomics, as you've already observed.

Reply:
Ah, yeah great point. I think `word` is definitely a better namespace over `vocabulary`, I'll switch over to that.